ggshield/cmd/main.py

```diff
@@ -100,6 +100,13 @@ def cli(
     load_dot_env()
 
     config = ctx.obj["config"]
+
+    if os.getenv("NO_COLOR"):
+        ctx.color = False
+    else:
+        if os.getenv("GL_USERNAME"):
+            ctx.color = True
+
```

Comment:
Nit: use a helper function to insulate the color logic
Nit2: add a comment on why you try to check the GL_USERNAME